src/web/pages/extras/cvssV4/Metrics.jsx

```diff
@@ -3,24 +3,28 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import _ from 'gmp/locale';
 import Select from 'web/components/form/Select';
+import useTranslation from 'web/hooks/useTranslation';
+
 
 const Metrics = ({metrics, selectedOptions, handleOptionChange}) => {
-  return Object.entries(metrics).map(([metricShort, metric]) => (
-    <section key={`${metric.name}-${metricShort}`}>
-      <Select
-        items={Object.entries(metric.options).map(([value, name]) => ({
-          label: `${_(name)} (${value})`,
-          value: value,
-        }))}
-        label={`${_(metric.name)} (${metricShort})`}
-        name={metricShort}
-        value={selectedOptions[metricShort]}
-        onChange={handleOptionChange}
-      />
-    </section>
-  ));
+  const [_] = useTranslation();
+  return Object.entries(metrics).map(([metricShort, metric]) => {
+    return (
+      <section key={`${metric.name}-${metricShort}`}>
+        <Select
+          items={Object.entries(metric.options).map(([value, name]) => ({
+            label: `${_(name)} (${value})`,
+            value: value,
+          }))}
+          label={`${_(metric.name)} (${metricShort})`}
```

Comment:
same here. metric.name will only get translated if the string is marked somewhere else for being translatable and therefore gets extracted into the json files.